app.js

```diff
@@ -66,6 +66,13 @@ app.use(compression())
 app.locals.GITHUB_SHA = process.env.GITHUB_SHA || null
 app.locals.hasData = hasData
 
+// Using the getClientJs to grab the global app.js and set it in app.locals so we can render in templates
+const { getClientJs } = require('./utils')
+app.use(function(req, res, next) {
+  app.locals.appJs = getClientJs(req, 'app')
+  next()
+})
+
```

Comment:
I'm not crazy about this bit. Open to alternative suggestions.

The issue is, webpack is compiling all of our js/css assets using chunkhash filenames. For page-specific js, we have a helper that checks the manifest and pulls in relevant js files for the route. But what todo with a global app.js?

Reply:
Do we have to add vue app wide? Can we just keep it at page specific for now?

Reply:
see latest